tests/app/forms/validation/test_number_validator.py

```diff
@@ -1,124 +1,60 @@
-
-        mock_form = Mock()
-        mock_field = Mock()
-        mock_field.raw_data = [""]
-
-        with self.assertRaises(StopValidation) as ite:
-            validator(mock_form, mock_field)
-
-        self.assertEqual(error_messages["INVALID_NUMBER"], str(ite.exception))
-
-    def test_non_numeric_string_invalid(self):
-        validator = NumberCheck()
-
-        mock_form = Mock()
-        mock_field = Mock()
-        mock_field.raw_data = ["a"]
-
-        with self.assertRaises(StopValidation) as ite:
-            validator(mock_form, mock_field)
-
-        self.assertEqual(error_messages["INVALID_NUMBER"], str(ite.exception))
-
-    def test_numeric_exponential_invalid(self):
-        validator = NumberCheck()
-
-        mock_form = Mock()
-        mock_field = Mock()
-        mock_field.raw_data = ["2E2"]
-
-        with self.assertRaises(StopValidation) as ite:
-            validator(mock_form, mock_field)
-
-        self.assertEqual(error_messages["INVALID_NUMBER"], str(ite.exception))
-
-    def test_decimal_number_invalid(self):
-        validator = DecimalPlaces(2)
-
-        mock_form = Mock()
-        mock_field = Mock()
-        mock_field.raw_data = ["1.234"]
-
-        with self.assertRaises(ValidationError) as ite:
-            validator(mock_form, mock_field)
-
-        error_message = error_messages["INVALID_DECIMAL"] % {"max": 2}
-        self.assertEqual(error_message, str(ite.exception))
-
-    def test_space_invalid(self):
-        validator = NumberCheck()
-
-        mock_form = Mock()
-        mock_field = Mock()
-        mock_field.raw_data = [" "]
-
-        with self.assertRaises(StopValidation) as ite:
-            validator(mock_form, mock_field)
-
-        self.assertEqual(error_messages["INVALID_NUMBER"], str(ite.exception))
-
-    def test_zero_valid(self):
-        validator = NumberCheck()
-
-        mock_form = Mock()
-        mock_field = Mock()
-        mock_field.raw_data = ["0"]
-
-        try:
-            validator(mock_form, mock_field)
-        except StopValidation:
-            self.fail("Valid number raised StopValidation")
-
-    def test_positive_number_valid(self):
-        validator = NumberCheck()
-
-        mock_form = Mock()
-        mock_field = Mock()
-        mock_field.raw_data = ["10"]
-
-        try:
-            validator(mock_form, mock_field)
-        except StopValidation:
-            self.fail("Valid number raised StopValidation")
-
-    def test_negative_number_valid(self):
-        validator = NumberCheck()
-
-        mock_form = Mock()
-        mock_field = Mock()
-        mock_field.raw_data = ["-10"]
-
-        try:
-            validator(mock_form, mock_field)
-        except StopValidation:
-            self.fail("Valid number raised StopValidation")
+from app.forms.validators import DecimalPlaces
+
+
+@pytest.mark.parametrize(
+    "value",
+    (
+        [None],
+        [""],
+        ["a"],
+        ["2E2"],
+    ),
+)
+@pytest.mark.usefixtures("gb_locale")
```

Comment:
Any reason using `usefixture` ?

Reply:
Yes, when you need to run a test within the context of a fixture but don't need to use it directly in the test, use this instead to avoid having to add a linter ignore-this-unused-arg. It's only really needed if linting strictly, and we are